io.github.fastrizwaan.WineZGUI.yml

```diff
@@ -0,0 +1,156 @@
+app-id: io.github.fastrizwaan.WineZGUI
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.winehq.Wine
+base-version: stable-23.08
+command: winezgui
+#rename-icon: winezgui
+#rename-desktop-file: winezgui.desktop
+finish-args:
+  - --allow=devel # For Wine crash handling
+  - --allow=multiarch
+  - --device=all
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib32/gstreamer-1.0:/app/lib/gstreamer-1.0:/usr/lib/i386-linux-gnu/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0
+  - --env=WINEDLLPATH=/app/dlls/lib32:/app/dlls/lib:/app/lib32/wine/wined3d:/app/lib/wine/wined3d
+    # Need to link .desktop files
+  - --filesystem=~/.local/share/applications:create
+    # Desktop shortcut runs wine script from host
+  #- --talk-name=org.freedesktop.Flatpak
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=fallback-x11
```

Comment:
```
{
    "errors": [
        "finish-args-fallback-x11-without-wayland"
    ]
}
```

Does it support wayland? Otherwise just use x11

Reply:
wine currently only uses x11